### PR TITLE
Add stargazers and fork count

### LINKED ISSUE #7: Add stargazers and fork count for each repo
## Description

I like to see the number of stars and forks for my repositories, just like what's seen in Github Profile picture.

screen shot 2017-10-15 at 10 24 40 am

Use case is to simulate these cards in other websites.

### PR DESCRIPTION
**Pull request by mathdroid**
## Purpose
Include the stargazers count and forks count for each repo in the repository list.

Closes #7.

## Approach
Add additional parameter to the graphql query

#### Open Questions and Pre-Merge TODOs
- [x] Change the tests
- [x] Update `example-response.json`
- [x] Update `readme.md`

### CODE REVIEW

**Comment by sindresorhus** [comment]
👍 Makes sense. In addition to a test, the readme and https://github.com/sindresorhus/gh-latest-repos/blob/master/example-response.json needs to be updated too.

File: readme.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 I currently use this on [my website](https://sindresorhus.com/#projects).
 
-It returns the 6 latest repos and it's cached for a day.
+It returns the 6 latest repos along with their number of stars and forks. The result is cached for a day.
```
Comment (sindresorhus) [inline readme.md]: > It returns the 6 latest repos along with some metadata.

File: example-response.json [hunk 2]
```diff
@@ -1,58 +1,89 @@
...
-		{
+		"primaryLanguage": {
 			"name": "Swift",
 			"color": "#ffac45"
+		},
+		"stargazers": {
+			"totalCount": 34
+		},
+		"forks": {
+			"totalCount": 0
```
Comment (sindresorhus) [inline example-response.json]: I think the response should simply be:

```json
"stargazers": 34,
"forks": 0
```

**Comment by mathdroid** [comment]
Some tests were failed.
screen shot 2017-10-15 at 1 45 28 pm

File: index.js [hunk 3]
```diff
@@ -54,7 +60,11 @@ async function fetchRepos() {
 		token
 	});
 
-	const repos = body.user.repositories.nodes;
+	const repos = body.user.repositories.nodes.map(repo => repo && ({
```
Comment (sindresorhus) [inline index.js]: I don't think you need the `repo &&`. If there are no repos, `.map()` will not run.

**Comment by mathdroid** [comment]
Since we modify the response from github on our `fetchRepos()` function, we will need to 'revert' the result from `nock` to use the default-shaped response. This will make us have 2 jsons, one for our fixture, one for the github's fixture.

**Comment by sindresorhus** [comment]
Looks good. Thanks :)

**Pull request merged**